src/d3a/models/strategy/market_maker_strategy.py

```diff
@@ -30,8 +31,9 @@ def __init__(self, energy_rate_profile=None, energy_rate=None, grid_connected=Tr
         if energy_rate_profile is not None:
             energy_rate = read_and_convert_identity_profile_to_float(energy_rate_profile)
         else:
-            energy_rate = ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE \
-                if energy_rate is None else read_and_convert_identity_profile_to_float(energy_rate)
+            energy_rate = read_arbitrary_profile(
+                InputProfileTypes.IDENTITY, ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE
+            ) if energy_rate is None else read_and_convert_identity_profile_to_float(energy_rate)
```

Comment:
Hmm, here I am not sure, how about using `read_and_convert_identity_profile_to_float` for both cases?
```suggestion
            energy_rate = read_and_convert_identity_profile_to_float(ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE
            if energy_rate is None else energy_rate)
```